### PR TITLE
Adding version to names of static files

### PR DESCRIPTION
**Pull request by agaoglu**
to prevent browsers using old files in local caches after an upgrade.

Not sure if i should include new bindata along with this though.

refs #2379 

ping @juliusv

### CODE REVIEW

File: web/web.go [hunk 1]
```diff
@@ -425,6 +425,7 @@ func tmplFuncs(consolesPath string, opts *Options) template_text.FuncMap {
 		},
 		"consolesPath": func() string { return consolesPath },
 		"pathPrefix":   func() string { return opts.ExternalURL.Path },
+		"buildVersion": func() string { return opts.Version.Version },
```
Comment (juliusv) [inline web/web.go]: `Version.Revision` (or the first 7 characters of that) would be even better for people not building off a release version.

**Comment by juliusv** [comment]
Thanks! I think this is generally fine as a first step, but it makes me think that we should switch to some more proper asset pipline-y approach where we incorporate the hash of a file's contents into the filename.

**Comment by beorn7** [comment]
Also see #2411 .

**Comment by agaoglu** [comment]
I thought if there was a hash stored along with the file, there'd be a way to set it as a cachebuster, but it seems bindata has no such feature.

File: web/web.go (hunk 1 above)
Comment (agaoglu) [inline web/web.go]: Done

**Comment by juliusv** [comment]
:+1: Cool, thanks! Merging for now. If people want to build a more involved solution later, that's still possible, as it won't break anything.

**Comment by brian-brazil** [comment]
This broke the default console templates, as the buildVersion function isn't available there.

**Comment by juliusv** [comment]
@brian-brazil Oops, sorry. Thanks for noticing. Fix proposal in https://github.com/prometheus/prometheus/pull/2562

**Pull request merged**